charts/rollups-validator-node/values.yaml

```diff
@@ -56,7 +56,7 @@ global:
     # -- Global Docker image registry
     registry: docker.io
     # -- Global Docker Image tag
-    tag: 0.9.1
+    tag: 1.0.0
```

Comment:
this should be at https://github.com/cartesi/helm-charts/pull/10/commits/9eca37d66a46838ab2189b89d86c61879a3a1c3c

if you sent this as a fixup commit, I could reuse it

I'll fix it at the original PR